docs/how-tos/deploy-ml.md

```diff
@@ -66,9 +134,8 @@ For an object detector, use bounding boxes.
 
 You can use tags to create classification models for images.
 For example, if you would like to create a model that identifies an image of a star in a set of images, tag each image containing a star with a `star` tag.
-The filter also needs to include images without the star tag or with another tag like `notstar`.
-If you add a `notstar` tag, you can filter the data in your dataset by selecting `star` and `notstar` from the **Tags** dropdown in the **Filtering** menu.
-Alternatively you can use date ranges to filter for relevant data.
+You also need images without the star tag or with another tag like `notstar`.
+If you add a `notstar` tag.
```

Comment:
incomplete sentence. Not sure if you want to keep the whole original sentence or what is meant to be here.